parts/k8s/kubernetesprovisionsource.sh

```diff
@@ -96,4 +96,36 @@ systemctl_restart() {
             sleep $wait_sleep
         fi
     done
+}
+docker_health_probe()
+{
+  # finds out if docker runtime is misbehaving
+  every=10 #check every n seconds
```

Comment:
Did these values come from experience with testing, or just seem like reasonable choices?

Reply:
I ran `shellcheck` on this script just for grins, and it mostly complained about variable quoting so :+1:.